Helper/Store.php

```diff
@@ -92,4 +102,12 @@ public function getUserStores(): array
         $storeManagerDataList = $this->storeManager->getStores();
         return $this->parseStores($storeManagerDataList);
     }
+
+    /**
+     * get the current store from the storeManager
```

Comment:
Method description should start with a capital letter, and docblock should have a \@return